backend/translation/views.py

```diff
@@ -1,199 +1,246 @@
 from io import StringIO
 
-import requests
 import webvtt
 from django.shortcuts import get_object_or_404
 from drf_yasg import openapi
 from drf_yasg.utils import swagger_auto_schema
 from rest_framework import status
-from rest_framework.decorators import api_view
-from rest_framework.permissions import IsAuthenticatedOrReadOnly
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import IsAuthenticated
 from rest_framework.response import Response
-from rest_framework.views import APIView
 from transcript.models import Transcript
 
 from .metadata import INDIC_TRANS_SUPPORTED_LANGUAGES
-from .models import Translation
+from .models import * 
```

Comment:
Not an immediate thing to fix, but in-general, it is not a good practice to do `import *`.
We'll try to avoid it in future :)

Reply:
Yeah, yeah. It was actually done in Shoonya so I thought it's being practiced here. I personally also don't like `import *`. I will fix it.